.ci/build-mac-arm64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh -ex
+# llvm@16 doesnt exist for arm64 we will use the latest available
+brew install -f --overwrite nasm ninja git p7zip create-dmg ccache pipenv llvm@13 sdl2 glew cmake
+brew link -f llvm@13
+#export MACOSX_DEPLOYMENT_TARGET=12.0
+export CXX=clang++
+export CC=clang
+
+export BREW_PATH;
+BREW_PATH="$(brew --prefix)"
+export BREW_BIN="/usr/local/bin"
```

Comment:
Since this is the arm build, the brew path is opt/homebrew

Reply:
Thank you , i mostly use macports (lets me install universally ) so i forgot 😅